include/robot_body_filter/RobotBodyFilter.h

```diff
@@ -282,11 +289,13 @@ class RobotBodyFilter : public ::robot_body_filter::FilterBase<T> {
    *                            point captured at different time, it also needs
    *                            a float32 "stamps" channel and viewpoint
    *                            channels vp_x, vp_y and vp_z.
+   * \param sensorFrame Sensor frame id
    * \param mask Output mask of the points. Indices to the mask are taken from
    *             the INDEX channel.
    * \return Whether the computation succeeded.
    */
-  bool computeMask(const sensor_msgs::PointCloud2& projectedPointCloud, std::vector<RayCastingShapeMask::MaskValue>& mask);
+  bool computeMask(const sensor_msgs::PointCloud2& projectedPointCloud, const std::string& sensorFrame,
+                   std::vector<RayCastingShapeMask::MaskValue>& mask);
```

Comment:
Nitpick: i'd move the new parameter to the 3rd position, so that this method's signature is similar to that of `maskContainmentAndShadows()`.